plugins/woocommerce-beta-tester/includes/class-wc-beta-tester-live-branches.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Beta Tester Plugin Live Branches feature class.
+ *
+ * @package WC_Beta_Tester
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Beta_Tester Live Branches Feature Class.
+ */
+class WC_Beta_Tester_Live_Branches {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'register_scripts' ) );
+
+		// By the time this code runs it appears too late to hook into `admin_menu`.
+
+		// NOTE - We don't have feature flags, so add the following code to enable it
+		// in development: `$this->register_page()`.
+	}
+
+	/**
+	 * Register live branches scripts.
+	 */
+	public function register_scripts() {
+		if ( ! method_exists( 'Automattic\WooCommerce\Admin\Loader', 'is_admin_or_embed_page' ) ||
+			! \Automattic\WooCommerce\Admin\Loader::is_admin_or_embed_page()
+		) {
+			return;
+		}
```

Comment:
This block should use `PageController` to prevent this deprecation warning

```
Deprecated: Function is_admin_or_embed_page is deprecated since version 6.3! Use \Automattic\WooCommerce\Admin\PageController::is_admin_or_embed_page() instead. in /var/www/html/wp-includes/functions.php on line 5383
```

```php
if ( ! method_exists( 'Automattic\WooCommerce\Admin\PageController', 'is_admin_or_embed_page' ) ||
		! \Automattic\WooCommerce\Admin\PageController::is_admin_or_embed_page()
) {
		return;
}
```

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/35643/commits/888a82ec4c9d0cc84f13037fef4a2437efa5e024